src/app/qgscustomization.h

```diff
@@ -104,6 +104,7 @@ class APP_EXPORT QgsCustomizationDialog : public QMainWindow, private Ui::QgsCus
 
     QString mLastDirSettingsName;
     QSettings *mSettings = nullptr;
+    QList<QWidget *> mSelectedWidgets;
```

Comment:
Let's be super-safe:

```suggestion
    QList<QPointer< QWidget > > mSelectedWidgets;
```